proposals/infra/INF-NNNN-default-function-inlining.md

```diff
@@ -0,0 +1,97 @@
+<!-- {% raw %} -->
+
+# Default Function Inlining
+
+* Proposal: [INF-NNNN](INF-NNNN-default-function-inlining.md)
+* Author(s): [Greg Roth](https://github.com/pow2clk)
+* Sponsor: [Greg Roth](https://github.com/pow2clk)
+* Status: **Under Consideration**
+* Impacted Project(s): DXC, DXC-Clang
```

Comment:
I think we just say "Clang" for the Clang project, unless "DXC-Clang" is referring to something more specific?

Reply:
Is this proposing changes to DXC as well?

Reply:
I'm not proposing changes to DXC, though this largely documents how things are in DXC without much consideration of changes in clang just yet. I would prefer a more specific term than "clang" to refer to the modernized compiler, but I won't fight that battle now.